openhtf/plugs/usb/__init__.py

```diff
@@ -36,56 +38,153 @@ def MyPhase(test, adb):
 conf.Declare('usb_server', 'USB Server IP/Hostname')
 conf.Declare('usb_server_port', 'USB Server Port',
              default_value=10000)
-
 conf.Declare('libusb_rsa_key', 'A private key file for use by libusb auth.')
+conf.Declare('device_type', 'android, fastboot or general usb type')
+conf.Declare('usb_hub', 'local or Ethersyncy')
```

Comment:
Talked in person and leaning towards a ```remoteusb``` config key. If that key isn't set, assume local USB. If it is set, it will tell us the type of remote USB connection to make (ethersync, minnodw, etc.).